kernel/classes/ezrssexport.php

```diff
@@ -449,6 +449,10 @@ function generateFeed( $type )
                 $creatorObject->attribute('name'), ENT_NOQUOTES, 'UTF-8'
             );
         }
+        else
+        {
+            $author->name = "Admin from " . $config->variable( 'SiteSettings', 'SiteName' );
```

Comment:
Actually this was more exemple than anything... 

I'm not even sure if using "Admin" is good.
I've thought of using :
- the Lastname/Firstname of the admin user
- the AdminMail from the site.ini 

Using the AdminMail would duplicate `$author->email `, but, if you consider that this is more a workaround than a bugfix... well, I think that we won't find something which will fit for all the cases (AdminMail from .ini is often fake as well as the AdminUser which is not always related to a real person...).

What do you think ?